lib/cryptoexchange/exchanges/exx/services/market.rb

```diff
@@ -0,0 +1,50 @@
+module Cryptoexchange::Exchanges
+  module Exx
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Exx::Market::API_URL}/tickers"
+        end
+
+        def adapt_all(output)
+          output.map do |key, value|
+            base, target = key.split('_')
+            market_pair = Cryptoexchange::Models::MarketPair.new(
+              base:   base,
+              target: target,
+              market: Exx::Market::NAME
+            )
+            adapt(value, market_pair)
+          end
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Exx::Market::NAME
+          ticker.ask       = NumericHelper.to_d(output['sell'])
+          ticker.bid       = NumericHelper.to_d(output['buy'])
+          ticker.last      = NumericHelper.to_d(output['last'])
+          ticker.high      = NumericHelper.to_d(output['high'])
+          ticker.low       = NumericHelper.to_d(output['low'])
+          ticker.volume    = NumericHelper.to_d(output['vol'])
+          ticker.timestamp = Time.now.to_i
```

Comment:
@Swingcloud in your comment the API seem to return a date, is that something we can use for timestamp?